CvGameCoreDLL_Expansion2/CvPlayer.cpp

```diff
@@ -31129,6 +31129,24 @@ int CvPlayer::GetHappinessFromMilitaryUnits() const
 	return iCount / iHappyPerXUnits;
 }
 
+int CvPlayer::GetYieldFromMilitaryUnits(YieldTypes eIndex) const
```

Comment:
Thank you for taking the time to make this a yield-agnostic ability instead of culture-specific